tools/cli/commands/docker.js

```diff
@@ -745,6 +759,17 @@ export function dockerDefine( yargs ) {
 						} ),
 					handler: argv => execDockerCmdHandler( argv ),
 				} )
+				.command( {
+					command: 'phpunit-wpcomsh',
+					alias: 'phpunit:wpcomsh',
```

Comment:
Given this alias has never existed, perhaps we don't need it?

Reply:
🤷